bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -38,9 +38,14 @@ class LayoutDOMView extends BokehView
       _.delay(resize, 100)
       _.delay(resize, 200)
 
+    if @model.responsive is 'fixed'
```

Comment:
I'd rather not. The correct javascript is `===`. Coffeescript compiles both `==` and `is` to `===`. At least this way I'm not polluting my muscle memory.